tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -391,6 +393,14 @@ class RegularExpressionTranspilerSuite extends FunSuite with Arm {
     doTranspileTest("a\\Z{1,}", "a(?:[\n\r\u0085\u2028\u2029]|\r\n)?$")
   }
 
+  test("transpile predefined character classes") {
+    doTranspileTest("\\p{Lower}", "[a-z]")
+    doTranspileTest("\\p{Alpha}", "[a-zA-Z]")
+    doTranspileTest("\\p{Alnum}", "[a-zA-Z0-9]")
+    doTranspileTest("\\p{Punct}", "[!\"#$%&'()*+,\\-./:;<=>?@\\^_`{|}~\\[\\]]")
+    doTranspileTest("\\p{Print}", "[a-zA-Z0-9!\"#$%&'()*+,\\-./:;<=>?@\\^_`{|}~\\[\\]\u0020]")
+  }
```

Comment:
Is it intentional that only a subset are tested here?

Reply:
Yes, this one is mostly a sanity check since the `\p{Print}` transpilation recursively uses the definition of the 4 `\p{}` classes above it. 
I tested the full set in the integration tests.